test/test_linearizer.py

```diff
@@ -440,6 +442,7 @@ def test_local_and_grouped_reduce(self):
       [Opt(OptOps.LOCAL, 0, 2), Opt(OptOps.GROUPTOP, 0, 2)],
       [Opt(OptOps.LOCAL, 0, 16), Opt(OptOps.GROUPTOP, 0, 16)],
       [Opt(OptOps.LOCAL, 0, 32), Opt(OptOps.GROUPTOP, 0, 2)],
+      [Opt(OptOps.LOCAL, 0, 32), Opt(OptOps.GROUPTOP, 0, 2), Opt(OptOps.UNROLL, 0, 0)],
```

Comment:
Is 0, 0 even valid?

Reply:
It is - but sorry I didn't mean to have this here... the error I wanted to test is down in the new `test_color_shapes_with_local`.  Will fix and push.